packages/dashboard/src/features/database/pages/IndexesPage.tsx

```
@@ -26,30 +66,332 @@ interface IndexRow extends DataGridRowType {
+        unique,
+        concurrently,
+      });
+      onOpenChange(false);
+    } catch {
+      // keep dialog open; caller reports errors via toast
+    } finally {
+      setIsCreating(false);
+    }
+  };
+
+  const isValid = !!tableName && !!indexName.trim() && selectedColumns.length > 0 && !indexNameError;
```

ESLint

[error] Insert `⏎···`

(prettier/prettier)